engine/verification/match/engine.go

```diff
@@ -408,25 +407,8 @@ func (e *Engine) requestChunkDataPack(c *ChunkStatus) error {
 		Nonce:   rand.Uint64(), // prevent the request from being deduplicated by the receiver
 	}
 
-	// find other execution nodes
-	others, err := e.state.Final().
-		Identities(filter.And(
-			filter.HasRole(flow.RoleExecution),
-			filter.Not(filter.HasNodeID(c.ExecutorID, e.me.NodeID()))))
-	if err != nil {
-		return fmt.Errorf("could not find other execution nodes identities: %w", err)
-	}
-
-	targetIDs := []flow.Identifier{c.ExecutorID}
-
-	// request chunk data pack from another execution node if exists as backup
-	if len(others) > 0 {
-		other := others.Sample(1).NodeIDs()[0]
-		targetIDs = append(targetIDs, other)
-	}
-
 	// publishes the chunk data request to the network
-	err = e.con.Publish(req, targetIDs...)
+	err := e.con.Publish(req, c.ExecutorID)
```

Comment:
We only request chunk data packs from original executor

Reply:
I'm a bit concerned that if the original executor was down, then all VNs that have received that receipts will not be able to receive the chunk data pack?

Should we add some redundancy? For instance, another EN who produced the same result?

Reply:
won't this happen automatically - ie other EN broadcast its own ER ?

Reply:
I think the redundancy should only be called after X tries.

Reply:
yeah, makes sense

Reply:
Per conversation with @zhangchiqing we keep querying the original executor on current version. However, in new version of VNs that soon will be in place, we have a more advanced retry mechanism including the redundancy and backoff. 

Just a note that at the _current version_, we deduplicate results at `finder` engine, so at the `match` engine, we do not keep requesting the same chunk data from several executors. Rather, `match` engine keeps asking from the _first_ executor of a particular result that it receives its receipt. This is not the case in the new version of VNs.